### ISSUE TITLE
Stole

### ISSUE THREAD

**Issue by Benifix04** [nako75/xclash#1]
You are stolen repo

**Comment by nako75** [comment]
yeah, but i was added some description in readme.md 'Thanks to XeonObject for this project.'  xD
